tests/test_pyglmnet.py

```diff
@@ -388,3 +385,47 @@ def test_random_state_consistency():
 
     assert_array_equal(ypred_a, ypred_b)
     assert_array_equal(ypred_b, ypred_c)
+
+
+@pytest.mark.parametrize("distr", ALLOWED_DISTRS)
+def test_simulate_glm(distr):
+    """Test that every generative model can be simulated from."""
+
+    n_samples, n_features = 10, 3
+
+    # sample random coefficients
+    beta0 = np.random.normal(0.0, 1.0, 1)
+    beta = np.random.normal(0.0, 1.0, n_features)
+
+    np.random.seed(1)
```

Comment:
why not use random_state in simulate_glm? It's the preferred approach

Reply:
Changed it now. Can you please review how I do it and check if it's correct?